pkg/commands/execution_context.go

```diff
@@ -1387,6 +1400,9 @@ func (ctx *executionContext) findTests() error {
 	if len(ctx.tests) == 0 {
 		return errors.New("there is no tests defined")
 	}
+	if len(ctx.tests) < ctx.cloudTestConfig.FailedTestsLimit {
+		return errors.New("number of tests is less than the failed tests limit")
```

Comment:
Why treat this check result as an error, does it invalidates the testing or makes it irrelevant?
May be logrus.Warning() would be enough?

Reply:
It doesn't make any sense to add failedTestLimit greater than total number of tests because it is the same as running cloudtest without failedTestLimit option enabled at all

Reply:
> it is the same as running cloudtest without failedTestLimit option enabled at all

Still, it doesn't seem a sound reason to reject testing at all. 
The inconvenience might be a case when user willing to limit testing to few tests ('only-run' directive or via command line) will have to adjust failedTestLimit too.
failedTestLimit is a 'safety' measure to avoid wasting time when things go ugly and most of the tests fail.

Reply:
Sounds reasonable to me, seems that failedTestLimit is more cloudTest optimization than a testing option

Reply:
I worry that the warning in logs could be a not good notification for the user. 
@alex-yust Can we just ignore checking `failedTestLimitation`  in case if used `only-run` option?

Reply:
> Can we just ignore checking `failedTestLimitation` in case if used `only-run` option?

Sure we can, but it looks a bit as overthinking..
My personal choice would be to skip this check at all to keep things simple. The fact that number of tests found is less that the failedTestLimit doesn't seem important enough to report.

Reply:
I think we do not need to print anything in this case at all.